### DISCUSSION TITLE
Nuxt Bridge does not support definePageMeta

### DISCUSSION THREAD

**Discussion by MaksymTymoshenko2** [nuxt/bridge#913] [General]
Hello everyone, I've been considering transitioning from `Nuxt 2 `to `Nuxt 3 `using `Nuxt Bridge`. 
Most of the components have already been rewritten using the `script setup syntax`, and I don't have the ability to configure the config for each page, specifically to specify what type of `middleware`, `layout`, `etc`.
Is there any workaround for this, and when do the developers plan to add `definePageMeta` to the config?

**Comment by kurt-grung** [comment]
@neversaynever88 Same issue here, any workaround?
